### ISSUE TITLE
Consider making on-hover actions always visible

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#3198]
Currently when editing a dashboard, the `add series`, `remove card`, and `resize` actions are hidden until the user hovers their cursor over the card. This makes those actions inaccessible on tablets or other mobile devices.

If we intend on making Metabase usable on devices such as a Microsoft Surface or iPad, it seems to me it would make sense to have it be a principle that we don't hide functionality under hover events.

![screen shot 2016-08-15 at 3 21 35 pm](https://cloud.githubusercontent.com/assets/2223916/17681754/6f394e62-62fc-11e6-87d7-fd33bf443dbf.png)

![screen shot 2016-08-15 at 3 21 39 pm](https://cloud.githubusercontent.com/assets/2223916/17681755/6f3a5b54-62fc-11e6-833c-fee2990c46ec.png)

**Comment by mazameli** [comment]
An update on this: seems like we already are doing this on _some_ dashboard cards, but not consistently. Seems like it's working for pie charts, scalars, and multi-scalars, but not line or bar charts. All of these screenshots are from an iPhone 6 using the Chrome app:

![pie chart](https://cloud.githubusercontent.com/assets/2223916/17783779/5efbfd72-652e-11e6-9656-e728b2e5fb43.png)
![multi-scalar](https://cloud.githubusercontent.com/assets/2223916/17783775/5ef9ed3e-652e-11e6-81ca-61927b89a496.png)
![bar chart 2](https://cloud.githubusercontent.com/assets/2223916/17783776/5efa6a2a-652e-11e6-9f3f-60fd60754c68.png)
![bar chart](https://cloud.githubusercontent.com/assets/2223916/17783777/5efa8b22-652e-11e6-9f0c-ad66fe92d013.png)
![scalar chart](https://cloud.githubusercontent.com/assets/2223916/17783778/5efa892e-652e-11e6-92b5-7522782cf2d9.png)
![line chart](https://cloud.githubusercontent.com/assets/2223916/17783780/5efdac30-652e-11e6-8743-d3a6d7456d65.png)

**Comment by jonathankau** [comment]
I can confirm that I'm seeing this issue in Chrome on Mac OSX 10.10.5. None of the card types that I've tried thus far show the hover actions.

<img width="1246" alt="screen shot 2016-08-18 at 12 28 11 pm" src="https://cloud.githubusercontent.com/assets/4529259/17787763/5752af9c-653f-11e6-9da6-1e9f45694eb1.png">

**Comment by tlrobinson** [comment]
@jonathankau You'll have to click the pencil icon to enter edit mode before you'll see the action icons.

**Comment by mazameli** [comment]
Yeah, I'm strictly talking about mobile in this issue.

**Comment by jonathankau** [comment]
@tlrobinson Ah gotcha. Well I can confirm that the action icons don't appear for line charts or bar charts on Chrome for desktop.

**Comment by mazameli** [comment]
Click the pencil and then hover over the card. :) This is the intended behavior currently for desktops; this issue is proposing we consider changing this current behavior.

**Comment by jonathankau** [comment]
Yep, even after clicking on the pencil and hovering over cards, I still find that some cards don't display action buttons.

See screen recording: https://www.dropbox.com/s/5q2jwebit7hjuso/metabase_bug.mov?dl=0

**Comment by mazameli** [comment]
Oh, hmm. @tlrobinson is this that issue you just fixed?
